### PR TITLE
Deprecate Skins

### PR DESCRIPTION
**Pull request by hghazzi**
### This PR will...

Deprecate all skins, including seven, as well as the `skin.name` config

### Why is this Pull Request needed?

We are no longer going to support any JW7 in JW8

### Are there any Pull Requests open in other repos which need to be merged with this?

https://github.com/jwplayer/jwplayer-commercial/pull/3757

#### Addresses Issue(s):

JW8-86

### CODE REVIEW

**Comment by robwalch** [comment]
test this please

File: src/css/skins/seven.less [hunk 1]
```diff
@@ -1,71 +0,0 @@
-@import "../shared-imports/vars";
-@import "../controls/imports/transitions";
-
-.jw-skin-seven {
-    @display-bkgd-color: @controlbar-background;
-
-    .jw-display-icon-container {
-        border-radius: 3.5em;
```
Comment (robwalch) [inline src/css/skins/seven.less]: You might consider moving some of these styles to the appropriate class defaults (like styles in display.less). Just make sure you're only adding or replacing (not duplicating) a style like `border-radius`.

File: src/css/skins/seven.less [hunk 2]
```diff
@@ -1,71 +0,0 @@
...
-
-        & > .jw-icon {
-            color: rgba(255, 255, 255, 0.9);
```
Comment (robwalch) [inline src/css/skins/seven.less]: default

File: src/css/skins/seven.less [hunk 3]
```diff
@@ -1,71 +0,0 @@
...
-        }
-    }
-
-    .jw-dock-button {
-        // Seven skin dock buttons do not highlight, they only show the tooltip
-        &:hover {
-            background: @controlbar-background;
-        }
-
-        border-radius: 2.5em;
```
Comment (robwalch) [inline src/css/skins/seven.less]: default

File: src/css/skins/seven.less [hunk 4]
```diff
@@ -1,71 +0,0 @@
...
-    .jw-skip {
-        border-radius: @ui-padding;
-    }
-
-    .jw-text {
-        text-rendering: optimizeLegibility;
-        /* font-smoothing is a private setting not transformed by PostCSS */
-        -webkit-font-smoothing: antialiased;
-        /* OSX Firefox specific style for improving legibility */
-        -moz-osx-font-smoothing: grayscale;
```
Comment (robwalch) [inline src/css/skins/seven.less]: default

File: src/templates/error.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
-export default (id, skin = '', title = '', body = '') => {
+export default (id, title = '', body = '') => {
```
Comment (egreaves) [inline src/templates/error.js]: What happens if there is a custom skin that has styles for the error template?

Comment (robwalch) [inline src/templates/error.js]: I think we want to rename this one setup-error.js. I don't think it's necessary for us to apply the custom skin class here. There is not default so it doesn't belong in the template.

**Pull request merged**